easybuild/easyblocks/generic/cmakemake.py

```diff
@@ -317,9 +317,60 @@ def configure_step(self, srcdir=None, builddir=None):
                 self.cfg['configopts']])
 
         (out, _) = run_cmd(command, log_all=True, simple=False)
+        self.check_python_paths()
 
         return out
 
+    def check_python_paths(self):
+        """Check that there are no detected Python paths outside the EB installed PythonF"""
+        if not os.path.exists('CMakeCache.txt'):
+            self.log.warning("CMakeCache.txt not found. Python paths checks skipped.")
+            return
+        cmake_cache = read_file('CMakeCache.txt')
+        if not cmake_cache:
```

Comment:
Hmm, I don't see how this can happen, since at this point you know that the file exists?

Reply:
Yes, but it could be empty for some reason